app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -16,6 +16,10 @@
 final class CreateProfileHelper {
 
     private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int GOOD_FIELD = 100;
+    private static final int EMPTY_FIELD = 101;
+    private static final int TOO_SHORT_FIELD = 102;
```

Comment:
This seems to be duplicate code from the CreateProfileActivity.java file, can you in any way avoid using it/Why do you need it?